amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -67,6 +87,16 @@ function submitClean(){
 	document.aimAuditLoggerManagerForm.submit();
 }
 
+function submitFilter() {
+	if (document.getElementById("userId").value != null) {

```

Comment:
while submitting you just need to submit the full form and test in the action if selectedUser, selectedTeam is present, then use to filter in the general loading action (also submit filters while paginating)